astro/src/content/docs/apis/_user-login-response.mdx

```diff
@@ -1,29 +1,47 @@
 import APIField from 'src/components/api/APIField.astro';
 import InlineField from 'src/components/InlineField.astro';
 
+<APIField name="emailVerificationId" type="String">
+  An email verification Id. When present, this will represent the user's current email verification Id.
+
+  This field is only returned when the <InlineField>emailConfiguration.verificationStrategy</InlineField> is set to `FormField`.
+</APIField>
+
+<APIField name="identityVerificationId" type="String" since="1.99.9">
+  An identity verification Id.
+
+  This field is only returned under the following conditions, based on the identity that was used to log in:
+
+  * An email address - the <InlineField>emailConfiguration.verificationStrategy</InlineField> is set to `FormField`.
+  * A phone number - the <InlineField>phoneConfiguration.verificationStrategy</InlineField> is set to `FormField`.
+</APIField>
+
 <APIField name="pendingIdPLinkId" type="String" since="1.28.0" renderif={!!props.idp_response}>
-  The pending identity provider link Id. This value is created when logging in with an identity provider configured with a linking strategy of <code>Create a pending link</code>. It will only be included in the response body when this strategy is configured and a link does not yet exist for the user. It is used in conjunction with the <a href="/docs/apis/identity-providers/links">Link APIs</a> to complete a pending link.
+  The pending identity provider link Id. This value is created when logging in with an identity provider configured with a linking strategy of `Create a pending link`. It will only be included in the response body when this strategy is configured and a link does not yet exist for the user. It is used in conjunction with the [Link APIs](/docs/apis/identity-providers/links) to complete a pending link.
 </APIField>
 
-  <APIField name="refreshToken" type="String">
+<APIField name="refreshToken" type="String">
   The refresh token that can be used to obtain a new access token once the provided one has expired.
 
   Because a refresh token is per user and per application, this value will only be returned when an <InlineField>applicationId</InlineField> was provided on the login request and the user is registered to the application.
 
   You must explicitly allow generation of refresh tokens when using the Login API.
 
-  Configure the <InlineField>application.loginConfiguration.generateRefreshTokens</InlineField> setting via the API or enable the setting by navigating to the <strong>Application -&gt; My Application -&gt; Security</strong> tab.
+  Configure the <InlineField>application.loginConfiguration.generateRefreshTokens</InlineField> setting via the API or enable the setting by navigating to the **Application -> My Application -> Security** tab.
```

Comment:
Should be using `<Breadcrumb>foo -> bar -> baz</Breadcrumb>`

Reply:
Done